libs/styles/styles.css

```diff
@@ -742,7 +742,7 @@ header.global-navigation + .feds-localnav {
   height: var(--global-height-navPromo);
 }
 
-.feds-sr-only {
+sr-only, .feds-sr-only {
```

Comment:
this should move to global.css.js from MAS otherwise, outside Milo, `sr-only` will be shown to MAS consumers.

Reply:
But not all pages have global.css.js either.
For example: https://main--cc--adobecom.aem.live/products/photoshop/plans?milolibs=MWPW-162956
<img width="211" alt="image" src="https://github.com/user-attachments/assets/80b9c7a5-7013-464e-91bf-f5054c233a47" />

Do you have a better place to add it?
(Maybe I should add both places?)

Reply:
You are right, global.css.js comes with a merch card.
As you suggested, let's keep it both in styles.css and in global.css.js